### PR TITLE
Some integration tests

### PR DESCRIPTION
**Pull request by adinaned**

### CODE REVIEW

File: src/DucksNet.API/DTO/UserDTO.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
-﻿namespace DucksNet.API.DTO;
+﻿using System.Xml.Linq;
```
Comment (Al-Andrew) [inline src/DucksNet.API/DTO/UserDTO.cs]: is this `using` needed?

File: src/DucksNet.API/DTO/PetDTO.cs [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿namespace DucksNet.API.DTO;
+﻿using DucksNet.Domain.Model;
+using System.Net;
```
Comment (Al-Andrew) [inline src/DucksNet.API/DTO/PetDTO.cs]: are this `using` needed?

File: tests/DucksNet.IntegrationTests/UsersControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        errors.Should().NotBeEmpty();
+        errors.Should().Contain("First name is required.");
+    }
+
+    [Fact]
+    public async Task When_CreateUserWithEmptyLastName_Then_ShouldFail()
+    {
+        ClearDatabase();
+        //Arrange
+        var sut = CreateInvalidSut("Aristo", " ", "Paris, Bonfamille's Residence", "0700000000", "les_aristocats@yahoo.com", "CatsAreTheBest");
```
Comment (Al-Andrew) [inline tests/DucksNet.IntegrationTests/UsersControllerTests.cs:62]: It's kinda funny that you created a helper function for making invalid objects but you still pass in all of the parameters for the constructor :)

**Review by Al-Andrew — APPROVED** [review]
:shipit:

File: tests/DucksNet.IntegrationTests/UsersControllerTests.cs (hunk 3 above)
Comment (adinaned) [reply tests/DucksNet.IntegrationTests/UsersControllerTests.cs:62]: Pai ca sa ocupe mai putine linii =))

**Pull request merged**